server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v64/DbVersion64Test.java

```diff
@@ -35,7 +35,7 @@ public void migrationNumber_starts_at_1600() {
 
   @Test
   public void verify_migration_count() {
-    verifyMigrationCount(underTest, 8);
+    verifyMigrationCount(underTest, 9);
```

Comment:
I think it's a good test as allow to fail if we remove a migration by mistake for instance